_posts/de/newsletters/2022-08-24-newsletter.md

```diff
@@ -0,0 +1,144 @@
+  [topic zero-conf channels] für ihren Node zu öffnen. Zu diese Funktionen
+  kommen zusätzlich viele andere neue Funktionen und Fehlerkorrekturen hinzu.
+
+- [LND 0.15.1-beta.rc1][] ist ein Release-Kandidat, der "Unterstützung
+  für [Zero-Conf-Kanäle][topic zero-conf channels] und scid-[Aliase][aliases]
+  enthält, [und der] überall zur Verwendung von [taproot]
+  [topic taproot]-Adressen wechselt".
+
+## Nennenswerte Code- und Dokumentationsänderungen
+
+*Erwähnenswerte Änderungen diese Woche in [Bitcoin Core][bitcoin core repo],
+[Core Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], und [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #25504][] ergänzt die Antworten auf `listsinceblock`,
+  `listtransactions`, und `gettransactions` um die zugehörigen
+  Deskriptoren in dem neuen Feld `parent_descs`. Zusätzlich, kann
+  `listsinceblock` nun angewiesen werden, explizit die Wechselgeld-Outputs mit
+  dem optionalen Parameter `include_change` aufzulisten. Normalerweise werden
+  Wechselgeld-Outputs als implizite Nebenprodukte von ausgehenden Zahlungen
+  weggelassen. Ihre Auflistung könnte im Kontext von Watch-Only-Deskriptoren
+  aber interessant sein.
+
+- [Eclair #2234][] fügt Unterstützung für DNS-Namen in Ankündigungen von Nodes
+  hinzu, wie es jetzt von [BOLTs #911][] erlaubt wird (siehe [Newsletter
+  #212][news212 bolts911]).
+
+- [LDK #1503][] fügt Unterstützung für [Onion-Nachrichten][topic onion messages]
+  hinzu, wie in [BOLTs #759][] definiert. Der PR deutet an, dass diese Änderung
+  eine Vorbereitung für die Unterstützung von [Offers][topic offers] ist, welche
+  später hinzugefügt werden kann.
+
+- [LND #6596][] fügt einen neuen `wallet addresses list`-RPC hinzu, der alle
+  Adressen eines Wallets und deren aktuelle Kontostände auflistet.
+
+- [BOLTs #1004][] empfiehlt, dass Nodes, welche routing-relevante Informationen
+  über Kanäle aufbewahren, mit dem Löschen dieser Informationen mindestens 12
+  Blöcke warten, nachdem ein Kanal geschlossen wurde. Die verzögerte Löschung
+  ermöglicht das Erkennen von [splices][topic splicing], worin ein Kanal nicht
+  wirklich geschlossen wird, sondern stattdessen weitere Mittel in einer
+  Onchain-Transaktion hinzugefügt oder entfernt werden.
+
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="25504,2234,1503,911,759,6596,1004" %}
+[core lightning 0.12.0]: https://github.com/ElementsProject/lightning/releases/tag/v0.12.0
+[bdk 0.21.0]: https://github.com/bitcoindevkit/bdk/releases/tag/v0.21.0
+[lnd 0.15.1-beta.rc1]: https://github.com/lightningnetwork/lnd/releases/tag/v0.15.1-beta.rc1
+[news212 bolts911]: /en/newsletters/2022/08/10/#bolts-911
```

Comment:
```suggestion
[news212 bolts911]: /de/newsletters/2022/08/10/#bolts-911
```

Reply:
See https://github.com/bitcoinops/bitcoinops.github.io/pull/851 which enables this link to work.

@tschweg take a look if you have a second

Reply:
@bitschmidty : Sorry for the late reply, I was out for a while. Thanks for catching the bad links! :+1: